latest/bpg/scalability/control-plane.adoc

```diff
@@ -313,4 +313,92 @@ If you call the API without any arguments it will be the most resource intensive
 /api/v1/pods
 ----
 
+=== Prevent DaemonSet thundering herds
+
+A DaemonSet ensures that all (or some) nodes run a copy of a pod. As nodes join the cluster, the daemonset-controller creates pods for those nodes. As nodes leave the cluster, those pods are garbage collected. Deleting a DaemonSet will clean up the pods it created.
+
+Some typical uses of a DaemonSet are:
+
+* Running a cluster storage daemon on every node
+* Running a logs collection daemon on every node
+* Running a node monitoring daemon on every node
+
+On clusters with thousands of nodes, creating or updating a DaemonSet can result in a high load placed on the control plane. If DaemonSet pods issue expensive API server requests on pod start-up, they can cause high resource use on the control plane from a large number of concurrent requests. 
+
+In normal operation, you can use a `RollingUpdate` to ensure a gradual rollout of new DaemonSet pods. With a `RollingUpdate` update strategy, after you update a DaemonSet template, the controller kills old DaemonSet pods and creates new DaemonSet pods automatically in a controlled fashion. At most one pod of the DaemonSet will be running on each node during the whole update process. You can perform a gradual rollout by setting `maxUnavailable` to 1, `maxSurge` to 0, and `minReadySeconds` to 60. 
+```
+minReadySeconds: 60
+strategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxSurge: 0
+    maxUnavailable: 1
+```
+
+A `RollingUpdate` ensures the gradual rollout of new DaemonSet pods if the DaemonSet is already created and has the expected number of `Ready` pods across all nodes. Thundering herd issues can result under certain conditions that are not covered by `RollingUpdate` strategies.
+
+==== Prevent thundering herds on DaemonSet creation
```

Comment:
Another scenario we should cover is that daemonset is already installed and nodes are scaled out.

Reply:
Good callout! Added another subsection for "Prevent thundering herds on node scale-outs".